spec/factories/user_pathways.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :user_pathway do
+    user { association :user }
```

Comment:
pathway { association :pathway } should be added

Reply:
done